RadixWallet/Features/AccountDetailsFeature/Coordinator/AccountDetails+Reducer.swift

```diff
@@ -128,8 +134,8 @@ public struct AccountDetails: Sendable, FeatureReducer {
 				// FIXME: Rename `ImportMnemonic` -> `ExportOrImportMnemonic` ?
 				ImportMnemonic()
 			}
-			Scope(state: /State.importMnemonics, action: /Action.importMnemonics) {
-				ImportMnemonicsFlowCoordinator()
+			Scope(state: /State.importMnemonic, action: /Action.importMnemonic) {
+				ImportMnemonicControllingAccounts()
```

Comment:
hmm this kinds of break something Matt has said in the past, to always trigger import of ALL missing mnemonics when once trigger import...

Reply:
@Sajjon Reverted back to use the whole import mnemonics flow.